trussworks/react-uswds
# fix: adding labels in combo box (default combo box, with default value)

# Summary
According to the USWDS docs the combo box should always have a label. Some of the examples in the storybook do not have labels. So the commit adds labels in the component file `src/components/forms/ComboBox/ComboBox.stories.tsx` to avoid confusion that they are optional.

## Related Issues or PRs
Closes #2608

## How To Test

1. Go to Default Combo box and With Default Value
2. Users can see that there is a label added to them

### Screenshots (optional)

![default](https://github.com/trussworks/react-uswds/assets/87698014/aff7b07e-6928-46ec-b3a1-483f598e18c0)
![with_default_value](https://github.com/trussworks/react-uswds/assets/87698014/22ea6130-7511-49e2-87ff-f5ce4a5ae0ee)
